extensions/podman/src/extension.ts

```diff
@@ -496,8 +503,11 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
     monitorMachines(provider);
   }
 
+  // update the status of the provider if the socket is changed, created or deleted
+  disguisedPodmanSocketWatcher = setupDisguisedPodmanSocketWatcher(provider, getSocketPath());
```

Comment:
Trying to get this to work but unable to. Seems that it runs once and then exits? Any tips @benoitf 

This is similar to the code here: https://github.com/containers/podman-desktop/blob/f40eab64c453e1bd5f562a61304eed28424d13dd/extensions/kube-context/src/extension.ts#L116